R/load_data.R

```diff
@@ -245,8 +245,9 @@ LoadSpatialCoordinates <- function (
       if (!requireNamespace("arrow", quietly = TRUE)) {
         abort(glue("Package {cli::col_br_magenta('arrow')} is required. Please install it with: \n",
                    "install.packages('arrow')"))
+      } else if (requireNamespace("arrow", quietly = TRUE)) {
+        coords <- arrow::read_parquet(file = coordinatefiles[i], as_data_frame = TRUE, mmap = TRUE)
       }
```

Comment:
I would move the `requireNamespace("arrow", quietly = TRUE)` out of the if statement. If the package is missing the function will abort. And if it's installed, using arrow:: will attach the function so no need to run requireNamespace. 
So I would switch this:
````
else if (requireNamespace("arrow", quietly = TRUE)) {
        coords <- arrow::read_parquet(file = coordinatefiles[i], as_data_frame = TRUE, mmap = TRUE)
      }
````
to simply use:

````
coords <- arrow::read_parquet(file = coordinatefiles[i], as_data_frame = TRUE, mmap = TRUE)
````

Reply:
Alright, thanks that makes sense. Then I'll go back to how it was before. We're still getting a warning about this though when you run the checks on the package, so there's something about how this is being called that the checks doesn't like.

Reply:
Sorry, I wasn't clear. This is what it should look like:
````
if (!requireNamespace("arrow", quietly = TRUE)) {
      abort(glue("Package {cli::col_br_magenta('arrow')} is required. Please install it with: \n",
                "install.packages('arrow')"))
} 
coords <- arrow::read_parquet(file = coordinatefiles[i], as_data_frame = TRUE, mmap = TRUE)
````
I meant to say that this part should be moved out: `coords <- arrow::read_parquet(file = coordinatefiles[i], as_data_frame = TRUE, mmap = TRUE)
`